python/lsst/dax/apdb/cassandra/apdbCassandra.py

```diff
@@ -449,12 +449,14 @@ def _get_version(key: str, default: VersionTuple) -> VersionTuple:
 
     @classmethod
     def apdbImplementationVersion(cls) -> VersionTuple:
```

Comment:
While most people will use the `Apdb` interface instead of the implementation classes, should this be renamed to `_apdbImplementationVersion` just in case? As it is, it will still show up in [the public docs](https://pipelines.lsst.io/v/daily/py-api/lsst.dax.apdb.ApdbSql.html#lsst.dax.apdb.ApdbSql.apdbImplementationVersion).

Reply:
Well, subclasses can add public methods, I see no problem with that. Most people will see `Apdb` and should need to look at implementaion classes.